src/riscv-codegen.c

```diff
@@ -152,6 +157,11 @@ void emit_ph2_ir(ph2_ir_t *ph2_ir)
     int rs2 = ph2_ir->src1 + 10;
     int ofs;
 
+    /* Prepare the variables to reuse the same code for
+     * the instruction sequence of division and modulo.
+     */
+    rv_reg soft_div_rd = __t0, soft_div_rs2_sign_reg = __t1;
```

Comment:
`soft_div_rs2_sign_reg` looks weird. Can you shorten it?

Reply:
How about renaming it as  `divisor_mask_reg`? 

At the beginning of the emulation, it obtains the absolute values of the dividend and divisor using the following approach.
```c
int32_t abs(int32_t x) 
{
    int32_t mask = x >> 31;       /* arithmetic right shift */
    return (x + mask) ^ mask
}
```
Therefore, the register  `__t1` will store `0` or `0xFFFFFFFF` initially, indicating that it serves as the mask for the divisor. 
(Similarly, `__t0` will be the mask for the dividend.)

If the requested operation is division, `divisor_mask_reg` will be `__t1` and helps to determine the result's sign. Otherwise,  for the modulo operation, because it only needs the dividend's mask to determine the result's sign, `divisor_mask_reg` becomes unnecessary and is assigned to `__zero` so that it doesn't affect the result's sign.

Reply:
Simply name it `divisor_mask` as it is intended to behave as its name suggests.